README.md

```diff
@@ -70,10 +70,23 @@ For Nano X:
 
 You can install the locally built binaries onto your physical Ledger Nano S, or Nano S Plus by running a flash script from *host*, not from *shell*. Note that Nano X does not support this.
 
-But before doing so you must install the prerequisites on your host. This requires Python3.
+### Prerequisites
 
+In order to allow sideloading, you must install the following prerequisites on your host. This requires Python3.
+
+Install python dependencies:
+```sh
+python3 -m pip config set global.break-system-packages true && pip3 install protobuf==3.20.3 && pip3 install ledgerwallet==0.5.1 && pip3 install ledgerblue
+```
+
+Install thumbv6m-none-eabi:
 ```sh
-python3 -m pip config set global.break-system-packages true && pip3 install protobuf==3.20.3 && pip3 install ledgerwallet==0.5.1
+rustup target add thumbv6m-none-eabi
```

Comment:
this is not what did it though, it was the binary file you downloaded right?

https://rdxworks.slack.com/archives/D035GDBE2NB/p1682506818826569